charts/csi-powerstore/Chart.yaml

```diff
@@ -1,21 +1,20 @@
 apiVersion: v2
-appVersion: "2.10.1"
+appVersion: "2.11.0"
 name: csi-powerstore
-version: 2.10.1
+version: 2.11.0
 description: |
   PowerStore CSI (Container Storage Interface) driver Kubernetes
   integration. This chart includes everything required to provision via CSI as
   well as a PowerStore StorageClass.
 type: application
-kubeVersion: ">= 1.23.0"
-# If you are using a complex K8s version like "v1.23.3-mirantis-1", use this kubeVersion check instead
-# WARNING: this version of the check will allow the use of alpha and beta versions, which is NOT SUPPORTED
-# kubeVersion: ">= 1.23.0-0"
+kubeVersion: ">= 1.24.0"
+# If you are using a complex K8s version like "v1.24.3-mirantis-1", use this kubeVersion check instead
+# kubeVersion: ">= 1.23.0-0
```

Comment:
Same comment as before-- if you're going to leave this comment we should at least close off the quotation marks.

Reply:
Fixed in #473